hologram_dashboard/fetcher/fetcher.cc

```diff
@@ -0,0 +1,5 @@
+#include "fetcher_lib/hologram_availability_fetcher.h"
```

Comment:
Name it fetcher_main.cc to make it more clear to readers.

Reply:
This is the library, the main would reside in fetcher.cc should i change fetcher.cc to fetcher_main instead?